xray/formats/table.go

```diff
@@ -52,11 +87,34 @@ type OperationalRiskViolationTableRow struct {
 	LatestVersion             string                       `col-name:"Latest\nVersion" extended:"true"`
 }
 
+type OperationalRiskViolationScanTableRow struct {
+	Severity string `col-name:"Severity"`
+	// For sorting
+	SeverityNumValue       int
+	DirectDependencies     []DirectPackagesTableRow `embed-table:"true"`
+	ImpactedPackageName    string                   `col-name:"Impacted\nPackage"`
+	ImpactedPackageVersion string                   `col-name:"Impacted\nPackage\nVersion"`
+	ImpactedDependencyType string                   `col-name:"Type"`
+	RiskReason             string                   `col-name:"Risk\nReason"`
+	IsEol                  string                   `col-name:"Is\nEnd\nOf\nLife" extended:"true"`
+	EolMessage             string                   `col-name:"End\nOf\nLife\nMessage" extended:"true"`
+	Cadence                string                   `col-name:"Cadence"  extended:"true"`
+	Commits                string                   `col-name:"Commits"  extended:"true"`
+	Committers             string                   `col-name:"Committers"  extended:"true"`
+	NewerVersions          string                   `col-name:"Newer\nVersions" extended:"true"`
+	LatestVersion          string                   `col-name:"Latest\nVersion" extended:"true"`
+}
+
 type DirectDependenciesTableRow struct {
 	Name    string `col-name:"Direct\nDependency"`
 	Version string `col-name:"Direct\nDependency\nVersion"`
 }
 
+type DirectPackagesTableRow struct {
```

Comment:
This struct and possibly other structs in this file can be made private. Let's check all of them.
The same goes for the fields on those structs.

Reply:
Done